python/cuml/benchmark/algorithms.py

```diff
@@ -69,6 +69,10 @@
     import umap
 
 
+if has_hdbscan_prediction(raise_if_unavailable=False):
```

Comment:
I think has_hdbscan makes more sense. Initially we only cared that the plotting package was available so it was named accordingly but since then we've added the prediction and we only really care that hdbscan itself is available.